modoboa_amavis/sql_email.py

```diff
@@ -41,7 +41,12 @@ def msg(self):
 
         if self._msg is None:
             mail_text = get_connector().get_mail_content(self.mailid)
-            self._msg = email.message_from_string(mail_text)
+            try:
+                self._msg = email.message_from_string(mail_text)
+            except UnicodeEncodeError:
+                # py2 expects bytes
+                mail_text = mail_text.encode('ascii', 'ignore')
+                self._msg = email.message_from_string(mail_text)
```

Comment:
Why not use the ``smart_str`` shortcut provided by django?
```python
from django.utils.encoding import smart_str
self._msg = email.message_from_string(smart_str(mail_text))
```

Reply:
I didn't think to check if django had any helper functions.

I've updated the pr with your suggestion.